carbon-design-system/carbon
# fix(uishell): add background & font color to active current links

Closes #4842 

#### Changelog

**New**

- add light gray background color to active/current page links
- add dark gray font color to active/current page links

**Removed**

- some duplicate/unnecessary styles

#### Testing / Reviewing

To test in the  `carbon-components-react` deployment --
1. Go to the `Fixed Sidenav` story
2. Inspect one of the last two side nav items "L0 Link"
3. In the inspector, find the `a` (link) node and add one of the following:
  - `aria-current="page"` attribute, OR
  - `.bx--side-nav__link--current` class

That way you can mimic an active link & see what happens when it gets an active/current ARIA or class.
